.grenrc.json

```
@@ -0,0 +1,8 @@
+{
+    "dataSource": "commits",
```

I believe `commits` as data source can be way too granular and will produce a lot of noise.
Perhaps PRs is a better option.
```suggestion
    "dataSource": "prs",
```

---

I tried this - but it did not produce meaningful output.  I've added an additional entry in the package.json so that it will create a draft release instead.  The notes can be edited or modified after generation.